style/activity.css

```diff
@@ -42,13 +42,38 @@
   top: 0px;
   background: var(--jp-layout-color0);
   z-index: 2;
+  display: flex;
+  justify-content: space-between;
 }
 
 .Verdant-events-date-header-label {
   margin-left: 1.5em;
   margin-top: 2px;
 }
 
+.Verdant-events-date-header-arrow {
+  display: inline-block;
+  vertical-align: text-bottom;
+  background-image: var(--verdant-icon-datearrow-open);
+  width: 1.4em;
+  height: 1.4em;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: 1.4em;
```

Comment:
At least on my browser, works a bit easier for sizing of the chevron if you keep width: 1.4em and then delete the other sizing rules "height" and "background-size". Check that this looks correct on your browser, but sometimes having fewer css rules work better.